packages/button-react/src/Button.test.tsx

```diff
@@ -1,11 +1,29 @@
 import React from "react";
 import { cleanup, render } from "@testing-library/react";
-import { PrimaryButton, SecondaryButton } from ".";
+import { PrimaryButton, SecondaryButton, TertiaryButton } from ".";
 
-describe("PrimaryButton", () => {
+describe("Button", () => {
     afterEach(cleanup);
 
-    it("should call the onClick handler when clicked", () => {
+    it("renders the correct button", () => {
+        const { getByText } = render(
+            <>
+                <PrimaryButton onClick={() => {}}>primary</PrimaryButton>
+                <SecondaryButton onClick={() => {}}>secondary</SecondaryButton>
+                <TertiaryButton onClick={() => {}}>tertiary</TertiaryButton>
+            </>,
+        );
+
+        const primaryButton = getByText("primary");
+        const secondaryButton = getByText("secondary");
+        const tertiaryButton = getByText("tertiary");
+
+        expect(primaryButton).toHaveClass("jkl-button--primary");
+        expect(secondaryButton).toHaveClass("jkl-button--secondary");
+        expect(tertiaryButton).toHaveClass("jkl-button--tertiary");
```

Comment:
I don't like multiple test in a test.

I'd prefer to split this into three test. Then each will report as its own test.

```
[
        { name: "Primary", component: PrimaryButton },
        { name: "Secondary", component: SecondaryButton },
        { name: "Tertiary", component: TertiaryButton },
    ].map((buttonVariant) => {
        it(`should work for ${buttonVariant}`, () => {
            const C = buttonVariant.component;

            const { getByText } = render(<C onClick={() => {}}>{buttonVariant.name}</C>);
            expect(getByText(buttonVariant.name)).toHaveClass(`jkl-button--${buttonVariant.name.toLowerCase()}`);
        });
    });
```

Reply:
Ooh, I like this!